docs/docs/monitoring/analytics.mdx

```diff
@@ -1,19 +1,876 @@
 ---
+id: analytics
 sidebar_label: Analytics
 title: Analytics
-description: Install Rasa Open Source on premises to enable local and customizable Natural Language Understanding and Dialogue Management.
+description:
+abstract: Rasa Pro Analytics is a data pipeline that extracts model, training,
+  and conversation data from the Rasa platform and streams the data to
+  your data warehouse for analysis.
 ---
 
-:::tip Rasa Pro License
-Rasa plus provides Enterprise-grade, flexible infrastructure and APIs, on top of Rasa, for enterprise scale, complexity, and interoperability.
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import RasaProLabel from "@theme/RasaProLabel";
+import RasaProBanner from "@theme/RasaProBanner";
 
-[Schedule a technical demo ->](https://info.rasa.com/rasa-technical-demo)
+<RasaProLabel />
+
+<RasaProBanner />
+
+Analytics helps visualize and process Rasa assistant metrics in the
+tooling (BI tools, data warehouses) of your choice. Visualizations
+and analysis of the production assistant and its conversations allow
+you to assess ROI and improve the performance of the assistant over time.
+
+<div align="center">
+  <img
+    alt="Rasa Pro Analytics example dashboard"
+    src={useBaseUrl("/img/analytics/analytics-examples.png")}
+    width="100%"
+  />
+</div>
+
+Measuring a Rasa assistant's success and making strategic investments
+will lead to better business outcomes. It will also help you understand
+the needs of your end users and better serve them.
+
+<div align="center">
+  <img
+    alt="An overview of the components of Rasa Pro."
+    src={useBaseUrl("/img/rasa_pro_overview.png")}
```

Comment:
I'd keep the schema, make it consistent with other schemas we have and rename to file to something like `rasa-pro-analytics-overview.png` or something like that